### PR TITLE
Update list of multiple properties

### PR DESCRIPTION
**Pull request by skjnldsv**
```php
in_array($property->name, ['URL', 'GEO', 'CLOUD', 'ADR', 'EMAIL', 'IMPP', 'TEL', 'X-SOCIALPROFILE', 'RELATED', 'LANG', 'X-ADDRESSBOOKSERVER-MEMBER'])
```

I really hate this!
I'm curious to see if we don't have a valid list somewhere which we wouldn't have to manually update :facepalm:

### CODE REVIEW

**Comment by nickvergessen** [comment]
Unit tests at https://github.com/nextcloud/server/blob/caff1023ea72bb2ea94130e18a2a6e2ccf819e5f/apps/dav/tests/unit/CardDAV/AddressBookImplTest.php#L357-L360 need adjusting

**Comment by skjnldsv** [comment]
Tests pushed with additional testing array :)

File: apps/dav/tests/unit/CardDAV/AddressBookImplTest.php [hunk 1]
```diff
@@ -355,8 +358,88 @@ public function testVCard2Array() {
 			],
 
 			'X-SOCIALPROFILE' => [
-				'twitter'=> 'tw-example',
-				'facebook'=> 'fb-example',
+				'tw-example',
+				'tw-example-2',
+				'fb-example',
```
Comment (nickvergessen) [inline apps/dav/tests/unit/CardDAV/AddressBookImplTest.php:363]:
```suggestion
				['type' => 'twitter', 'value' => 'tw-example'],
				['type' => 'twitter', 'value' => 'tw-example-2'],
				['type' => 'facebook', 'value' => 'fb-example'],
```

Should be it from reading the code

Comment (skjnldsv) [reply apps/dav/tests/unit/CardDAV/AddressBookImplTest.php:363]: Nope, only if you have specified the withType, that's why I added the seconds tests

Comment (nickvergessen) [reply apps/dav/tests/unit/CardDAV/AddressBookImplTest.php:363]: Ah, now I see it

**Review by ChristophWurst — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**